imblearn/under_sampling/prototype_generation/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+The :mod:`imblearn.under_sampling.prototype_generation` submodule
+contains the method in which a new samples are generated such that the
+dataset become more balanced.
+"""
```

Comment:
I would write something like that: The :mod:`imblearn.under_sampling.prototype_generation` submodule
contains methods that generate new samples in order to balanced the dataset.